bubbles-map/src/MapBubble.tsx

```diff
@@ -24,19 +30,28 @@ export class MapBubble extends Component<LatLongProps, State> {
     this.state = {
       latitude: 28.7041,
       longitude: 77.1025,
+      titleenabled: false,
       title: "Most Populous Cities in Asia",
+      zoom: 1,
+      bubblecolor: "blue",
     };
     
     this.receiveEvent = (event: any) => {
       const params = event.data.properties as Map<string, object>;
       const latitude = params.get("latitude");
       const longitude = params.get("longitude");
+      const titleenabled = params.get("titleenabled");
       const title = params.get("title");
+      const zoom = params.get("zoom");
+      const bubblecolor = params.get("bubblecolor");
 
       this.setState({
         latitude: params.get("latitude") as any,
         longitude: params.get("longitude") as any,
+        titleenabled: params.get("titleenabled") as unknown === "false",
         title: params.get("title") as any,
+        zoom: params.get("zoom") as any,
+        bubblecolor: params.get("bubblecolor") as any,
```

Comment:
Please notice that you take these params two times, I woudl recommend you yo use the bubblecolor variable (also for the others variables as well. This way you can make validations over the params and modify the values if needed, for example:

* Check if some value came from BC for bubble color, if not, use a default value, something like this:
```
const bubblecolor = params.get("bubblecolor") ? "#" +params.get("bubblecolor")  : "blue" ;
```